src/workloads/execution/GeoHaystackIndex.yml

```diff
@@ -0,0 +1,39 @@
+# This workload inserts ~1GB of documents in the first phase, and creates a geoHaystack index on them in the second.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 500000
+    BatchSize: 10000
+    Document:
+      a: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
```

Comment:
I wonder if we can pull some of this common code out using some of the Genny helpers added for this. @rtimmons thoughts?